subprojects/docs/src/docs/release/notes.md

```diff
@@ -381,8 +381,13 @@ testLauncher.withTestsFor(spec -> {
 ```
 
 
+### IDE integration improvements
```

Comment:
```suggestion
```
Suggestion: No need for a new section, this fits just fine under the "IDE" section.